Formula/apt.rb

```diff
@@ -0,0 +1,114 @@
+class Apt < Formula
+  desc "Advanced Package Tool"
+  homepage "https://wiki.debian.org/apt"
+  url "https://deb.debian.org/debian/pool/main/a/apt/apt_2.3.7.tar.xz"
+  sha256 "c946a126263b3ea51a15216689602367c0da95d6878b76dc891ea335d87bd936"
+  license "GPL-2.0-or-later"
+
+  depends_on "cmake" => :build
+  depends_on "docbook-xsl" => :build
+  depends_on "doxygen" => :build
+  depends_on "libxslt" => :build
+  depends_on "perl" => :build
+  depends_on "w3m" => :build
+
+  depends_on "berkeley-db"
+  depends_on "bzip2"
+  depends_on "docbook"
+  depends_on "dpkg"
+  depends_on "gcc"
+  depends_on "gettext"
+  depends_on "gnutls"
+  depends_on "googletest"
```

Comment:
Debian/Ubuntu list a `gpgv` dependency, which would be part of `gnupg` for us.
AUR package lists `gnupg` dependency.

I recall various GPG-related parts for `apt`, so want to check if it is needed or not.

Reply:
I added `gnupg`, and I also moved `googletest`, `docbook` and `po4a` to be build dependencies, as no binaries are linked to them.